landlab/components/plant_competition_ca/plant_competition_ca.py

```diff
@@ -67,38 +65,43 @@ class VegCA(Component):
     False
     """
 
-    _name = "Cellular Automata Plant Competition"
-
-    _input_var_names = (
-        "vegetation__cumulative_water_stress",
-        "vegetation__plant_functional_type",
-    )
-
-    _output_var_names = ("plant__live_index", "plant__age")
-
-    _var_units = {
-        "vegetation__cumulative_water_stress": "None",
-        "vegetation__plant_functional_type": "None",
-        "plant__live_index": "None",
-        "plant__age": "Years",
-    }
-
-    _var_mapping = {
-        "vegetation__cumulative_water_stress": "cell",
-        "vegetation__plant_functional_type": "cell",
-        "plant__live_index": "cell",
-        "plant__age": "cell",
+    _name = "VegCA"
```

Comment:
Is there a reason to change the name of the component to the name of the class? I kind of thought of the `_name` attribute as being a more descriptive name.

Reply:
You are right... I will change the places where I changed this back to their original state.